content/docs/user-guide/data/numpy.md

```diff
@@ -0,0 +1,33 @@
+# Numpy
+
+DataType, Reader and Writer implementations for `np.ndarray` and `np.number`
+primitives
+
+## Description
+
+**TODO**
```

Comment:
I see 11 TODOs in the changes.

Reply:
This is not getting to the website. These are just page stubs, without links in the sidebar.

Reply:
Can they be found by search engines btw?

Reply:
If they're not linked from anywhere, then they can't be found by crawlers.